join.html

```diff
@@ -6,7 +6,7 @@
 
 ## Join Us
 There are two simple requirements to join us: the first one is to live in Turin's area (so that you can come to our meetings), and the second one is to do one of the followings:
```

Comment:
I'd say we should just remove this page, and reimplement it when time will come :)

Reply:
did it